experimental/sdk/webpubsub-socketio-extension/src/SIO/components/web-pubsub-adapter.ts

```diff
@@ -72,8 +121,26 @@ export class WebPubSubAdapterInternal extends NativeInMemoryAdapter {
    * @param rooms - a set of rooms
    */
   public async addAll(id: SocketId, rooms: Set<Room>): Promise<void> {
-    // TODO: Implement this method.
-    // For now, it is intentionally empty.
+    // TODO: RT should support a new API AddConnectionsToGroups
+    debug(`addAll SocketId = ${id}, |rooms| = ${rooms.size}`);
+    try {
+      const eioSid = this._getEioSid(id);
+
+      // TODO: Temporary mitigation. This behaviour should be taken only one time and by service
+      rooms = rooms.add(""); // add namespace default room
+
+      for (const room of rooms) {
+        const groupName = this._getGroupName(this.nsp.name, room);
+        debug(
+          `Try to add connection ${eioSid} to group ${groupName}, convert from ns#room = ${this.nsp.name}#${room}, SocketId = ${id}`
+        );
+        
+        await this.service.group(groupName).addConnection(eioSid);
+      }
+    }
+    catch (error) {
+      console.error(error);
```

Comment:
also here, what is the best way for error handling?

Reply:
See [here](https://github.com/Azure/azure-webpubsub/pull/506#discussion_r1222919307). I prefer throwing all error to user.